### ISSUE TITLE
Bluetooth Keyboard & Mouse Support

### ISSUE THREAD

**Issue by PM250B** [2bt/GTMobile#14]
It's unclear whether there is significant demand for this, but it could be beneficial to allow users to control key functions using a Bluetooth keyboard and mouse. This might enhance accessibility and provide a more flexible user experience.

**Comment by 2bt** [comment]
Adding mouse support should be easy, I will check. Keyboard will be more work, probably not justified. Best accessibility is probably tablet + MIDI keyboard.
